### PR TITLE
docs(dev): Tighten up CONTRIBUTING and its linkage

### LINKED ISSUE #13: Redesign CONTRIBUTING so it has less reasons to change
CONTRIBUTING.adoc is the entry point for developers wanting to contribute to this project. It is a standard file with a standard name (CONTRIBUTING.*) and location (the root of the project). A project will have project specific developer documentation (e.g., how to run the automated tests) and project independent developer documentation (e.g., LFP workflow, etc.). With this in mind, we should design CONTRIBUTING.adoc so that it only has one reason to change. Right now it has too many reasons to change. This is going to cause conflicts when we update the project independent documentation.

So I propose the following:

* CONTRIBUTING.adoc - Contains important links to other documentation that will not change (LICENSE, DCO, etc.), including a link to a project specific documentation file (maybe docs/dev/project-specific.md).
* Moving workflow stuff to docs/dev/workflow/ .

### PR DESCRIPTION
**Pull request by StoneyJackson**
* Add links in CONTRIBUTING.md to all files in docs/dev/*
* Tweek and expand various sections

Closes: #13

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
 ## LibreFoodPantry
 
 This project is a [LibreFoodPantry](https://librefoodpantry.org) project.
-We are a member of this larger community and shares its resources
-and policies including its
+We are a part of this larger community and shares its policies and resources.
```
Comment (kwurst) [inline CONTRIBUTING.md]: I suggest "values, policies, and resources."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
...
+* Download and install git from [git-scm.com](https://git-scm.com/).
+* [Configure Git](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).
+
+When you configure Git, give the same name and email that you gave to GitHub
+and GitLab. Again, remember this name and email will become public, as git will
+include them in each commit you make, and when you contribute them, they
+will be publicly visible.
+
+Next, on GitHub or GitLab, __fork the project(s)__ you will be working on.
+If you are working in a shop, your shop manager will create a shop fork.
```
Comment (cradkowski) [inline CONTRIBUTING.md:104]: Maybe link to the shop setup documentation somewhere under here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
...
-* Use email for personal, private matters.
+We license source code under [GPL v3](LICENSE.md)
+and content under [GFDL v1.3](LICENSE_FOR_CONTENT.md).
+For each commit, contributors must
+* Include a [copyright notice](copyright-notices/) on each new file
+* Sign-off on the [DCO v1.1](DCO.txt) in the commit message
+* And credit all co-authors in the commit message
 
+Please read
+[Licensing](docs/dev/licensing.md)
```
Comment (cradkowski) [inline CONTRIBUTING.md:49]: This doesn't link to anything (no document named "licensing" in this folder).

**Review by kwurst — CHANGES_REQUESTED** [review]
I left one suggestion for a change. I feel like entirely removing Vision and Mission was a mistake. I think they can be subsumed under "values".

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
...
+* FOSSism #6: Embrace radical transparency
+* FOSSism #9: Keep a history
+* FOSSism #16: Avoid uncommunicated work
+
+We prefer public issue trackers over email because it is public
+and transparent. We prefer issue trackers over chat communication because
+it is more organized.
+
+For more information about our issue trackers, issue boards, issues, and labels
+please read [Issue Tracker](issue-tracker.md).
```
Comment (cradkowski) [inline CONTRIBUTING.md]: Doesn't link to anything, maybe:
docs/dev/issue-tracker.md

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -60,6 +110,8 @@ There are several types of issues in the issue tracker:
 Issues can and should be added at anytime during development.
 If you see something, say something.
 
+See also [Issue Tracker](issue-tracker.md).
```
Comment (cradkowski) [inline CONTRIBUTING.md]: Doesn't link, see previous comment above.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -123,6 +177,8 @@ the following questions.
   How much effort is required by each strategy? Can the issue be broken into
   smaller more manageable issues? If so, what would those smaller issues be?
 
+See also [Issue Tracker](issue-tracker.md).
```
Comment (cradkowski) [inline CONTRIBUTING.md]: Doesn't link, see previous comment above.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -92,6 +144,8 @@ This is an ongoing process. When you look through the issue tracker and its
 boards looking for something to work on, plan to comment on issues and help
 prioritize, organized, and prepare them for work.
 
+See also [Issue Tracker](issue-tracker.md).
```
Comment (cradkowski) [inline CONTRIBUTING.md]: Doesn't link, see previous comment above.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kwurst) [reply CONTRIBUTING.md]: And "share", not "shares".

**Review by cradkowski — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (StoneyJackson) [reply CONTRIBUTING.md]: Thanks! Fixing.

File: CONTRIBUTING.md (hunk 2 above)
Comment (StoneyJackson) [reply CONTRIBUTING.md:104]: Good idea. I'm adding it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (StoneyJackson) [reply CONTRIBUTING.md:49]: It should be there; I renamed docs/dev/copyright-licensing-and-dco.md → docs/dev/licensing.md . I was able to test this link by viewing the "code" tab and changing the branch to "contributing", then opening CONTRIBUTING.md and clicking on the link.

I'm marking this one resolved.

File: CONTRIBUTING.md (hunk 5 above)
Comment (StoneyJackson) [reply CONTRIBUTING.md]: This is real. Fixing it and the others.

Comment (StoneyJackson) [reply CONTRIBUTING.md]: Thanks for catching it!

**Comment by StoneyJackson** [comment]
Since I think I've addressed all of the issues you both raised (thanks again!), I'm going to use my superpowers to merge this PR. There are more changes I would like to make based on this, and I don't want to keep adding to this PR (since they are different issues).

**Pull request merged**